patches/chrome-installer-linux-common-google-chrome-google-chrome.appdata.xml.template.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/chrome/installer/linux/common/google-chrome/google-chrome.appdata.xml.template b/chrome/installer/linux/common/google-chrome/google-chrome.appdata.xml.template
+index 828dc1f0b625f411538751a8f8a12088aed7d729..b11609c809cc7931c512e3c30be911d82db66fa7 100644
+--- a/chrome/installer/linux/common/google-chrome/google-chrome.appdata.xml.template
++++ b/chrome/installer/linux/common/google-chrome/google-chrome.appdata.xml.template
+@@ -2,11 +2,11 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+   <id>@@PACKAGE@@.desktop</id>
+-  <update_contact>chromium-dev@chromium.org</update_contact>
++  <update_contact>support@brave.com</update_contact>
+   <metadata_license>CC0-1.0</metadata_license>
```

Comment:
NOTE: this `metadata_license` is for the actual XML file itself (not the project). More info available at https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html